packages/error-reporting/package.json

```diff
@@ -0,0 +1,55 @@
+{
```

Comment:
Overall, I'm still trying to figure out the integration goals, so I first assumed you were trying to work in isolation from our process. If the goal is for a tighter process integration, then we wouldn't skip yours, after this module's development workflow supports Windows developers.